tinygrad/runtime/ops_gpu.py

```diff
@@ -48,9 +48,11 @@ def synchronize(self):
 class CLBuffer(RawBufferCopyInOut, RawBufferTransfer):
   def __init__(self, size, dtype, device='0'): super().__init__(size, dtype, allocator=CL.cl_allocator, **{'device': device})
   def _copyin(self, x:np.ndarray):
+    if self.size == 0: return
```

Comment:
should these checks be in `RawBufferCopyInOut` instead? are they needed for the other backends?

Reply:
Yeah, that was the only backend to get segfault with size 0. But moved to Buffers, agree this is a bit cleaner.